app/src/main/java/org/mozilla/fenix/tabstray/AccessibleNewTabButtonBinding.kt

```diff
@@ -26,13 +27,14 @@ import org.mozilla.fenix.utils.Settings
 class AccessibleNewTabButtonBinding(
```

Comment:
We can always benefit with more @property documentations.

Reply:
I added some kdocs that are more meaningful than what we have now.